### ISSUE TITLE
Can't remove members

### ISSUE THREAD

**Issue by macrojames** [rhgrant10/Groupy#3]
The removal of members is not working.
I am guessing from the stacktrace you ran into:

    membership_id required
    string — Please note that this isn't the same as the user ID. In the members key in the group JSON, this is the id value, not the user_id.
```python
>>> k.remove(k.members()[6])
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/groupy/object/responses.py", line 272, in remove
    endpoint.Members.remove(self.id, member.user_id)
  File "/usr/local/lib/python3.4/dist-packages/groupy/api/endpoint.py", line 226, in remove
    return cls.response(r)
  File "/usr/local/lib/python3.4/dist-packages/groupy/api/endpoint.py", line 59, in response
    raise errors.ApiError(data['meta'])
groupy.api.errors.ApiError: {'errors': ['The requested path does not exist'], 'code': 404}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/groupy/object/responses.py", line 273, in remove
    except errors.InvalidResponse as e:
AttributeError: 'module' object has no attribute 'InvalidResponse'
>>> k.remove(groupy.User.get())
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/groupy/object/responses.py", line 272, in remove
    endpoint.Members.remove(self.id, member.user_id)
  File "/usr/local/lib/python3.4/dist-packages/groupy/api/endpoint.py", line 226, in remove
    return cls.response(r)
  File "/usr/local/lib/python3.4/dist-packages/groupy/api/endpoint.py", line 59, in response
    raise errors.ApiError(data['meta'])
groupy.api.errors.ApiError: {'errors': ['The requested path does not exist'], 'code': 404}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/groupy/object/responses.py", line 273, in remove
    except errors.InvalidResponse as e:
AttributeError: 'module' object has no attribute 'InvalidResponse'
```

**Comment by rhgrant10** [comment]
Hey thanks for the heads up. It looks from the output above that they have changed the URLs for the API. I'll look into it.

**Comment by rhgrant10** [comment]
As it turns out, there is still some code that uses the old exception class, but the main issue here is that the API requires the `id` of the user, which is unique in each group (aka the membership id).

So to successfully remove a member (for now) you'll need to pass in the member as returned from the group from which you're removing them.

I'll look into things further and at least release a new version with the correct exception names soon. After that I'll see what I can do about having it automatically grab the correct information since that should be possible.

Thanks again for the heads up :-)

**Comment by rhgrant10** [comment]
This should be fixed in v0.6.1, now available on the PyPI. Please let me know if you're still experiencing any issues.

**Comment by rhgrant10** [comment]
Fixed in  12f9996faf3f946185c45c222ac240782f855c5f.
